docs/UserGuide.md

```diff
@@ -237,14 +241,28 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find n/Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
-### Locating contacts by any field, similar to a search function: `find`
+### Searching people by any field: `find`
+
+Finds all persons based on the specified keywords after the prefix representing the field, and displays them as a list with index numbers.
 
-Finds all persons based on the specified keywords (case-insensitive) after the prefix representing the field, and displays them as a list with index numbers.
+Format: `find PREFIX_TAG KEYWORD [KEYWORD]…​`
 
-Format: `find PREFIX KEYWORD [KEYWORD]…​`
+<box type="tip" seamless>
+
+**Tip:** 
+* Available prefixes that can be used as the `PREFIX_TAG`:
+    * n/ Name
+    * a/ Address
+    * p/ Phone Number
+    * e/ Email
+    * t/ Tag
+    * w/ Wedding
+    * tk/ Task
+</box>
 
 * The search is case-insensitive. e.g `hans` will match `Hans`.
 * The prefix that corresponds to the field you want to search should be specified. e.g. use `find n/Alex` to search by name, use `find e/alex@gmail.com` to search by email.
+* The keyword after the prefix must be specified and cannot be empty. e.g. `find a/` is not allowed as the address keyword to search should be specified.
```

Comment:
Sorry to only note this here, but should this be a planned enhancement? It might indeed be a feature flaw in that you cannot search for blank values in fields that actually do support blank values (e.g., phone, address, etc.)

Reply:
Okay I agree will add it in